app/transport/rest/server.go

```diff
@@ -49,4 +49,11 @@ func (srv *Server) Run() error {
 func attachUserEndpoints(router *mux.Router, handlers Handlers) {
 	router.Path("/users").Methods(http.MethodPost).Handler(handlers.UserHandler.CreateUser())
 	router.Path("/users/{id}").Methods(http.MethodGet).Handler(handlers.UserHandler.GetUserByID())
+
+	var allowedSortArgs = strings.Join([]string{firstName, lastName, createdAt}, "|")
```

Comment:
Not required: `allowedSortArgs :=` this form of declaration is better
I recommend yo use var when you are ok with it's default value and modifying it later.
If you adding non-default value immediately - it's better to use `:=`.

Reply:
You are right! We do not want our Go  to look like JS 😄